client/sections/lib/a.sections.module.coffee

```diff
@@ -1,9 +1,25 @@
 angular.module 'sections', ['ui.router']
 
 .config ($stateProvider) ->
-    $stateProvider.state 'sections',
+    $stateProvider.state('section',
+        url: '/section'
         parent: 'root.main'
-        url: '/sections'
+        abstract: true
         templateUrl: 'client/sections/sections.html'
-        controller: 'SectionCtrl'
-        controllerAs: 'sectionCtrl'
+        controller: 'SectionsCtrl'
+        controllerAs: 'sectionsCtrl'
+    ).state('section.list',
+        url: 's'
+        templateUrl: 'client/sections/list/list.sections.html'
+    ).state('section.one',
+        # ten stan jest abstrakcyjny, ponieważ jego potomkiem jest oprócz section.one.show także
+        # topic i topic.new w module topics
+        url: '/:section_id'
+        abstract: true
+        templateUrl: 'client/sections/sections.html'
+    ).state('section.one.show',
+        url: ''
+        templateUrl: 'client/sections/show/show.section.html'
+        controller: 'ShowSectionCtrl'
+        controllerAs: 'showSectionCtrl'
```

Comment:
Ogólnie to controllerAs powinno być krótkie, ale też dość mocno opisowe. Ja bym proponował skrócić to do showSecCtrl na przykład albo shSecCtrl. Chyba, że to Ci nie przeszkadza.